modin/config/envvars.py

```diff
@@ -23,7 +23,7 @@
 from packaging import version
 from pandas.util._decorators import doc  # type: ignore[attr-defined]
 
-from .pubsub import (
+from modin.config.pubsub import (
```

Comment:
```suggestion
from modin.config import (
```
Does this work?

Reply:
Such imports will lead to cyclical imports.